Doc/whatsnew/3.8.rst

```diff
@@ -83,6 +83,29 @@ See :pep:`572` for a full description.
 .. TODO: Emily will sprint on docs at PyCon US 2019.
 
 
+Positional-only parameters
+--------------------------
+
+There is new syntax (``/``) to indicate that some function parameters
```

Comment:
Maybe "PEP 570 adds new syntax..." instead of "There is new syntax"?

"There is new syntax" is also fine grammatically, but usually the voice of these "What's new" entries seems to assume the context is that you are explaining what this PEP changes. "There is new syntax" feels every so slightly off because it de-emphasizes the change aspect of it.

I'll note that to me this is a very subtle difference and if I were not reading this specifically with giving notes on the wording in mind I would probably not have batted an eye, so if you prefer the original wording that's fine.

Reply:
I will leave  `There is new syntax` for symmetry with the assignment expressions paragraph.